### PR TITLE
Change cache parent

### PR DESCRIPTION
**Pull request by Baileyeatspizza**
During my testing when the Cache parent is set to the CurrentCamera the CurrentCamera would randomly change while in studio between causing the old camera to be parented to nil and thus the parts can't be seen this just changes the parent to Terrain an instance that can't be removed

sorry if I over explained this was a pretty much invisible and confused me for a while as when the cache made new parts they would be parented to the correct Camera

TLDR; Use Terrain as the cache's parent as Terrain can't be removed

### CODE REVIEW

**Comment by SamyBlue** [comment]
Yeah I agree. This might even lead to less side-effects given that Terrain can't be removed. Good catch
